src/Math/babylon.math.ts

```diff
@@ -1083,16 +1083,17 @@
             var nbRevert = 0;
             var cross: Vector3;
             var dot = 0.0;
+            var epsilon = 0.001;
```

Comment:
the first reason is that I didn't know it existed
the second reason is that I just arbitrary chose this value, but maybe 
it will need to be tuned (or not) to 0.00001 or to 0.01... I need 
(bugged) feedback to set it

but you're right, I will use Engine.Epsilon instead

Le 01/09/2015 16:25, Raanan Weber a écrit :
>
> In src/Math/babylon.math.ts 
> <https://github.com/BabylonJS/Babylon.js/pull/678#discussion_r38424075>:
>
> > @@ -1083,16 +1083,17 @@
> >              var nbRevert = 0;
> >              var cross: Vector3;
> >              var dot = 0.0;
> > +            var epsilon = 0.001;
>
> Why don't you use Engine.epsilon?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BabylonJS/Babylon.js/pull/678/files#r38424075>.
>